src/components/header/HeaderBody01.jsx

```diff
@@ -19,25 +23,30 @@ export default function HeaderBody() {
   };
   return (
     <>
-      <ContainerTitle>
-        <Title>
-          pourquoi nous <TitleSpan>existons</TitleSpan>?
-        </Title>
-      </ContainerTitle>
-
-      <ContainerAuthorSentence>
-        <AuthorSentence>
-          "Atteindre le plus grand nombre en mettant
-        </AuthorSentence>
-        <AuthorSentence> la barre très haut" Agnès Varda</AuthorSentence>
-      </ContainerAuthorSentence>
-      <>
-        <ContainerMoreInformation visible={visible} onClick={onClick}>
-          <MoreInformationSentence>En savoir plus</MoreInformationSentence>
-        </ContainerMoreInformation>
-        {visible ? <Main /> : null}
-        {visible ? <Footer /> : null}
-      </>
+      <Page>
+        <BlockTitle>
+          <ContainerTitle>
+            <Title>
+              pourquoi nous <TitleSpan>existons </TitleSpan> ?
+            </Title>
+          </ContainerTitle>
+        </BlockTitle>
+        <BlockSentence>
+          <ContainerAuthorSentence>
+            <AuthorSentence>
+              "Atteindre le plus grand nombre en mettant
+            </AuthorSentence>
+            <AuthorSentence> la barre très haut" Agnès Varda</AuthorSentence>
+          </ContainerAuthorSentence>
+        </BlockSentence>
+        <>
```

Comment:
*Advice*
Outside of the "JSX wrapper" role, I discourage the use of `<>` (or `<React.Fragment>`, for this matter). It won't have any existence/meaning in your page, so it's utterly useless